toptal/picasso
# fix: fix shadows import and type

[FX-2656]

### Description

<img width="1040" alt="Screen Shot 2022-04-27 at 21 23 15" src="https://user-images.githubusercontent.com/7733167/165593965-1ebf15e4-f0e8-4182-98f2-559bd51b9e99.png">

Due to having this ☝️ error while running unit tests, I started researching and found out that there is no direct access to this type and values anymore:
`import muiShadows, { Shadows } from '@mui/material/styles/shadows'`

We were having this for v4 and there wasn't any problem:
`import muiShadows, { Shadows } from '@material-ui/core/styles/shadows'`

Apparently, they introduced new shadow values inside `@mui/system` but there is also no way to use them as we have right now.  I saw a similar issue about the direct import of a color [here](https://github.com/mui/material-ui/issues/27296) but they also changed the colors import so it is not the exact same case.

While searching, I saw [this solution](https://github.com/mui/material-ui/issues/8289) from one of the authors of MUI. I have been inspired by it and provided this solution. Again, totally open for a better solution.

### How to test

- you can now run test:unit and see that the error is gone and jest is arguing about snapshots only.

### Development checks


**Breaking change**

- [ ] codemod is created

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-2656]: https://toptal-core.atlassian.net/browse/FX-2656?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
